languages/zh-CN/README.md

```diff
@@ -0,0 +1,142 @@
+# CloudEvents 规范
```

Comment:
I guess we have a decision to make... should all CE docs (for all languages) be under `/cloudevents` or should we group all docs for each languages under `/languages/LANG` ? 

ie.
```
/README.md
/languages/README.md
/cloudevents/README.md
/cloudevents/languages/zn-cn/README.md
/cloudevents/languages/zn-cn/spec.md
/discovery/README.md
/discovery/languages/zn-cn/README.md
/discovery/languages/zn-cn/spec.md
```
vs
```
/README.md
/cloudevents/README.md
/discovery/README.md
/languages/zh-cn/README.md
/languages/zh-cn/cloudevents/README.md
/languages/zh-cn/cloudevents/spec.md
/languages/zh-cn/discovery/README.md
/languages/zh-cn/discovery/spec.md
```

I'm wondering if our new release process, which seems to be focused on releasing "directories" (sort of), would influence this decision.

Reply:
For the first situation, each directory has it's own sub-dir /languages/LANG, while the other grouping all docs for each languages under /languages/LANG.  The difference between these two situations reflects two different specification structure.
The first structure seems to treat each directory as an independent spec.
i.e. 
```
/cloudevents  dir represents the CloudEvents spec
/discovery dir represents the Discovery spec
```
If that's the structure we want to release, then it makes sense that each dir holds its language sub-dir.

The second structure seems to treat all directories as the whole CloudEvents spec.
```
/cloudevents  dir represents the CloudEvents core spec
/discovery dir represents the CloudEvents additional spec about discovery
```
For that structure, it seems better if we group all docs under /languages/LANG.

So it all depends on what kind of spec structure we want to release finally.

Reply:
I personally prefer the second one coz I think that the CloudEvents spec should consist of core spec and other related specs.

Reply:
Or maybe we can re-org the structure like this:
```
/cloudevents
    /core    -> this core dir holds everything(like SDK.md, spec.md, adapters dir, etc) from the original /cloudevents dir
    /discovery   
    /subscription  
    /schemeregistry  
    /cesql 
    ...
/misc
/community
...
```

Reply:
Not sure if you were aware of the decision by the group but we decided to have the other spec dirs (eg. "discovery") hold all info for that spec. Basically like a nested git repo, but w/o the overhead of the repo. Then each release would be a 'tag' off of the 'main' branch with the dir being released as part of the tag name. Yes this means we'll be deleting the other branches and just have 'main' going forward. See: https://github.com/cloudevents/spec/pull/968

So, your statement:
> The first structure seems to treat each directory as an independent spec.

is how I view the current direction of the group. But we can wait for other folks to chime in to see if they have a different interpretation.

Reply:
I went through the tag-based structure. It does make sense. I noticed it's approved on the 3/18 call. So is this direction settled or not?

Reply:
yes I just need to find the time to "make it so".

Reply:
Cool, I'll update the language spec then to ensure each directory has a "languages" sub-dir.